src/Sulu/Bundle/AdminBundle/Resources/js/components/Tabs/Tabs.js

```diff
@@ -30,39 +30,37 @@ class Tabs extends React.Component<Props> {
         small: false,
     };
 
-    @observable wrapperWidth: number = 0;
-    @observable tabsWrapperWidth: number = 0;
     @observable tabsWidth: number = 0;
+    @observable tabsContainerWrapperWidth: number = 0;
+    @observable tabsContainerWidth: number = 0;
 
     @observable tabWidths: Map<number, number> = new Map();
     @observable tabRefs: Map<number, ?ElementRef<'li'>> = new Map();
     @observable dropdownOpen = false;
-    @observable lastSelectedIndex: ?number;
+    @observable lastShownIndex: ?number;
```

Comment:
I was suggesting `lastShownIndex` before I really understood what it does... I think `lastSelectedIndex` would be better in that case. But let's maybe discuss that behavior with @chirimoya tomorrow, before you change anything in that regard again.